src/plugins/vis_types/gauge/public/convert_to_lens/configurations/metric.ts

```diff
@@ -14,15 +14,14 @@ import {
 } from '@kbn/visualizations-plugin/common';
 import { GaugeVisParams } from '../../types';
 
-export const getConfiguration = (
+export const getMetricConfiguration = (
```

Comment:
I think is better to have tick not linked with the bands, feels more natural and clean.

Reply:
Sorry, just circling back to this PR.

Can you elaborate what you mean here? This code line does not relate to your comment, are you speaking generally?

Reply:
I don't remember exactly but I think I was speaking about the migration from a agg base gauge to a lens gauge (not the one to a metric that looks like in this file).
In that case, I believe we should configure the automatic tick alignment by default and not a tick aligned with the bands.

Reply:
I believe that `Ticks on bands` is on by default only when you initially switch on the `Band colors` switch. but the conversion doesn't seems to enable that switch for the Agg-based Gauge, but it enables it to the agg-based Goal chart.